FuncGodot Manual/pages/tips_runtime_building.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <title>FuncGodot Manual</title>
+    <link rel="icon" type="image/x-icon" href="../images/godot_ranger.svg">
+    <link rel="stylesheet" href="../styles.css">
+</head>
+<body>
+    <div class="main">
+        <h1>Runtime Map Building</h1>
+        <p>There are times where you may want to build maps while in the middle of running your game, whether it's for loading levels from map files instead of scenes, procedural generation, or as a way to support user-made maps. Runtime map building can also be combined with map editor launch configurations to provide a largely seamless experience between making a map and seeing it in-game.</p>
+        <h2>Setting up Runtime Map Building</h2>
+        <p>To build a map at Runtime, start by either using a templated scene with an empty <i>FuncGodotMap</i> or instantiating one. Next you'll need to make sure to set either the <code>local_map_file</code> or <code>global_map_file</code> to your map's file path. Don't use <code>load()</code> here! That will be taken care of in the next step.</p>
+        <p>Once your map file path has been set, you can run the <code>verify_and_build()</code> method from the <a href="ref_func_godot_map.html" target="main">FuncGodotMap</a> node. When this is done, your map will be built just as it would have been by pressing the <i>Build</i> button in the Editor, along with all of the entity callbacks. Be aware that the <code>Engine.is_editor_hint()</code> conditional will return as false, in case you use it to control properties handling through the FuncGodot entity callbacks.</p>
+        <p>When the map builds, it will emit the signals <code>build_complete</code> or <code>build_failed</code>. You can use these signals to handle errors gracefully and to run any post-build code you need to run.</p>
+        <p>If using <a href="https://docs.godotengine.org/en/4.2/tutorials/3d/global_illumination/using_lightmap_gi.html" target="_blank">LightmapGI</a> for your global illumination solution, you can use the <i>FuncGodotMap</i> node's <code>unwrap_uv2()</code> method after receiving the <code>build_complete</code> signal. Once unwrapping has finished, the <i>FuncGodotMap</i> node will emit <code>unwrap_uv2_complete</code>.</p>
+        <blockquote><i>NOTE: Godot 4.2 does not currently support runtime lightmap baking or UV2 unwrapping in exported builds. For now, consider using a different lighting solution for runtime building.<br></i></blockquote>
```

Comment:
We are now on 4.3 but this hasn't changed yet - however it is in progress and [awaiting review](<https://github.com/godotengine/godot/pull/94965>)